rosco-web/src/main/groovy/com/netflix/spinnaker/rosco/controllers/BakeryController.groovy

```diff
@@ -78,12 +78,18 @@ class BakeryController {
     return baseImage
   }
 
-  @ExceptionHandler
+  @ExceptionHandler(BakeOptions.Exception)
   @ResponseStatus(HttpStatus.NOT_FOUND)
   Map handleBakeOptionsException(BakeOptions.Exception e) {
     [error: "bake.options.not.found", status: HttpStatus.NOT_FOUND, messages: ["Bake options not found. " + e.message]]
   }
 
+  @ExceptionHandler(Exception)
```

Comment:
It was for debugging.

 I actually think we should have better exception handling. I found that I got generic bad request messages with null message text until I added the above.